p2p/discovery/bucket.go

```diff
@@ -30,24 +36,67 @@ func (b *bucket) addNode(node *Node) {
 		// do nothing for now
 		// TODO lru
 	} else {
+		b.lock.Lock()
+		defer b.lock.Unlock()
+
+		log.Error("add node: %s", hexutil.BytesToHex(node.ID.Bytes()))
```

Comment:
shoud be log.Info or Debug instead of Error.

Reply:
yes, forget to change